edx-olive/custom-registration-form
# [SE-4510] fix: update models to work with Koa

This registration form is not compatible with Koa. To make it compatible, we need to adjust the models and migrations.

**Dependencies**: None

**Testing instructions**:

1. Install the package on Koa using the readme of the repo (already installed [here](https://campus-koa.opencraft.hosting/register))
2. Check the "support education research..." checkbox
3. Validate the form that appeared
4. Check the record created in the database

**Author notes and concerns**:

N/A

**Reviewers**
- [ ] @nizarmah / @clemente
